edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -64,8 +100,27 @@ Seed cluster:: Dedicated {sno} cluster that you use to create a seed image and i
 
 {lcao}:: Generates the seed image, embeds the seed image URL in the live installation ISO, and reconfigures the host during remote site deployment.
```

Comment:
Can we double check on that.
The embedding of the seed image URL is done using the openshift-install command
The reconfiguration is done with ibio

Reply:
I believe what is technically correct is the following:

- the LCA operator generates the seed image
- the [lca-cli](https://github.com/openshift-kni/lifecycle-agent/tree/main/lca-cli) (which is part of the LCA operator codebase) is the software that actually reconfigures the SNO, which is included in the installation ISO (via the `openshift-install`, [here](https://github.com/openshift/installer/blob/master/data/data/imagebased/files/usr/local/bin/install-rhcos-and-restore-seed.sh#L16) is where it's called)
- the embedding of the seed image URL to the installation ISO is done via the `openshift-install` create installation ISO [here](https://github.com/openshift/installer/blob/master/data/data/imagebased/files/usr/local/bin/install-rhcos-and-restore-seed.sh#L5)
- the `openshift-install` or IBIO is the software creating the configuration ISO

I guess here we should just mention that the LCA operator is responsible for generating the seed OCI image. WDYT?